release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -123,7 +123,7 @@ In {product-title} 4.8, you can use Fujitsu hardware and the Fujitsu iRMC base b
 
 * Stopping the provisioning services once the installer deploys the control plane on the bare metal nodes. See link:https://bugzilla.redhat.com/show_bug.cgi?id=1949859[BZ#1949859] for more information.
 
-* Adding an Ironic health check to the bootstrap `keepalived` checks. See link:https://bugzilla.redhat.com/show_bug.cgi?id=1949859[BZ#1949859] for more information.
+* Adding an Ironic health check to the bootstrap Keepalived checks. See link:https://bugzilla.redhat.com/show_bug.cgi?id=1949859[BZ#1949859] for more information.
```

Comment:
Without other data, I'd revert this change because of the guidance on styling package names.